src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+            az container create -g MyResourceGroup --name MyAlpine --location westus --image
+            alpine:latest --cpu 1 --memory 1
+
+        Create a container group with OS type.
+            az container create -g MyResourceGroup --name MyWinApp --location westus --image
+            winappimage:latest --os-type Windows
+
+        Create a container group with public IP address.
+            az container create -g MyResourceGroup --name MyAlpine --location westus --image
+            alpine:latest --ip-address public
+
+        Create a container group with starting command line.
+            az container create -g MyResourceGroup --name MyAlpine --location westus --image
+            alpine:latest --command-line "/bin/sh -c '/path to/myscript.sh'"
+
+        Create a container group with envrionment variables.
+            az contanier create -g MyResourceGroup --name MyAlpine --location westus --image
+            alpine:latest -e key1=value1 key2=value2
+
+        Create a container group using container image from Azure Container Registry.
+            az container create -g MyResourceGroup --name MyAlpine --location westus --image
+            myAcrRegistry.azurecr.io/alpine:latest --image-registry-password password
+
+        Create a container group using container image from other private container image registry.
+            az container create -g MyResourceGroup --name MyApp --location westus --image myimage:latest
+            --cpu 1 --memory 1.5 --image-registry-login-server myregistry.com --image-registry-username
+            username --image-registry-password password
+
+Commands to get an Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container show: Show the details of a container group.
+
+    Arguments
+        --name -n           [Required]: The primary resource name.
+        --resource-group -g [Required]: Name of resource group. You can configure the default group
+                                        using `az configure --defaults group=<name>`.
+
+    Global Arguments
+        --debug                       : Increase logging verbosity to show all debug logs.
+        --help -h                     : Show this help message and exit.
+        --output -o                   : Output format.  Allowed values: json, jsonc, table, tsv.
+                                        Default: json.
+        --query                       : JMESPath query string. See http://jmespath.org/ for more
+                                        information and examples.
+        --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
+
+Commands to tail the logs of a Azure container group
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Command
+        az container logs: Tail the log of a container group.
+
+    Arguments
+        --name -n           [Required]: The primary resource name.
```

Comment:
What is the primary resource in this case?  The container?  The log? It is likely this should support `--ids` as well.